x/ethbridge/client/rest/rest.go

```diff
@@ -50,7 +50,7 @@ type burnOrLockEthReq struct {
 	EthereumReceiver  string       `json:"ethereum_receiver"`
 	Amount            sdk.Int      `json:"amount"`
 	Symbol            string       `json:"symbol"`
-	CethAmount        sdk.Int      `json:"ceth_amount" yaml:"ceth_amount"`
+	NativeTokenAmount sdk.Int      `json:"native_token_amount" yaml:"native_token_amount"`
```

Comment:
Can we add a REST route that gives the current lock and burn gas cost for a specific network as well as the currency needed to go to that network?

Reply:
It can help UI to get the config, will add the REST route for it.